File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.py
def solution(s):
    answer = 0
    arr = s.split(" ")
    
    for i in range(len(arr)) :
        if arr[i] == "Z" :
            temp = int(arr[i - 1])
            answer -= temp
        else : 
            temp = int(arr[i])
            answer += temp
        
    return answer

File: 프로그래머스/0/120853. 컨트롤 제트/컨트롤 제트.js
function solution(s) {
    var answer = 0;
    const arr = s.split(" ");
    
    for(let i in arr){
        if(arr[i] == 'Z'){
            answer -= parseInt(arr[i-1]);
        }else{
            answer += parseInt(arr[i]);
        }
    }
    return answer;
}